protocols/v2/binary-sv2/serde-sv2/src/primitives/sequences/seq064k.rs

```diff
@@ -9,6 +9,7 @@ use crate::{
 use alloc::vec::Vec;
 use serde::{ser, ser::SerializeTuple, Deserialize, Deserializer, Serialize};
 
+/// See `binary_sv2::Seq064k`
```

Comment:
Would this link to the `binary_sv2::Seq064k` in docs.rs ?

Reply:
@jbesraa, good eye/thought process. The answer is no, but check out [this comment](https://github.com/stratum-mining/stratum/discussions/970#discussioncomment-10630902) I left on the #970 discussion.

Let me know your thoughts on making this part of our documentation standard.